bin/index.cjs

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env node
+const chalk = require('chalk');
```

Comment:
Breaks due to dependencies not being installed prior to this point 😉

Reply:
Seems like dependencies used by the script have to be specified as regular deps, and not the dev ones.